libs/barista-components/input/src/input.scss

```
@@ -56,6 +56,12 @@
   }
 }
 
+.dt-focused {
```

This might be a bit dangerous, I'd suggest adding a .dt-form-field selector here as well. 
```suggestion
.dt-form-field.dt-focused {
```

---

Wasn't sure if `dtInput` should be aware of `dt-form-field`.
Maybe this should be placed in `dt-form-field` styles?

What do you think @ffriedl89 ?

---

Yeah i agree, it makes sense to include those styles in the form-field.scss instead.